app/pages/sections/catalogue-solutions/select-recipient/contextCreator.test.js

```diff
@@ -0,0 +1,38 @@
+import manifest from './manifest.json';
+import { getContext } from './contextCreator';
+import { baseUrl } from '../../../../config';
+
+jest.mock('../../getSectionErrorContext', () => ({
```

Comment:
don't think this needs mocked

Reply:
ooops already merged.. I'll remove for the next PR and then re-add it back in for the PR after that 😄